### PR TITLE
Replace `ex.` in guides with `for example`

### PR DESCRIPTION
**Pull request by bvhme**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/gh-pages/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

I was reading the guides using my screen reader and was perplexed when I heard 'ex-community'. After inspecting the letters I found the usage of a non-standard abbreviation for example with `ex`.

I personally feel writing out `for example` is a lot clearer than `ex.`.

Thanks so much, love the guides!

### CODE REVIEW

**Comment by nayafia** [comment]
Hi @bvhme, thanks for this suggestion! I like this idea, as I think it will make the guides less confusing for non-native English speakers in particular.

For readability purposes, could you add a comma after every `for example`?

> `for example, "We only spend about 5 hours per week on this project"`

Thanks!

**Comment by bvhme** [comment]
Yes, I've done it. Please check whether this is the consistency you were thinking of, after some Googling I found out that the comma should be there in case of a list, so I've added it where there were multiples.

**Comment by bvhme** [comment]
Can this be merged?

**Comment by mlinksva** [comment]
@bvhme I think it reads better with a comma after every "for example", and Googling around, it seems there isn't a hard rule to follow. "for example passwords or other non-public information" seems to be missed even following list rule.

File: _articles/en-US/best-practices.md [hunk 1]
```diff
@@ -67,8 +67,8 @@ Here are a few rules that are worth writing down:
 
 * How a contribution is reviewed and accepted (_Do they need tests? An issue template?_)
 * The types of contributions you'll accept (_Do you only want help with a certain part of your code?_)
-* When it's appropriate to follow up (_ex. "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread."_)
-* How much time you spend on the project (_ex. "We only spend about 5 hours per week on this project"_)
+* When it's appropriate to follow up (_for example "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread."_)
```
Comment (MikeMcQuaid) [inline _articles/en-US/best-practices.md]: Needs a comma

File: _articles/en-US/best-practices.md [hunk 2]
```diff
@@ -67,8 +67,8 @@ Here are a few rules that are worth writing down:
...
+* How much time you spend on the project (_for example "We only spend about 5 hours per week on this project"_)
```
Comment (MikeMcQuaid) [inline _articles/en-US/best-practices.md]: Needs a comma

File: _articles/en-US/how-to-contribute.md [hunk 3]
```diff
@@ -484,7 +484,7 @@ If the project is on GitHub, here's how to submit a pull request:
 
 * **[Fork the repository](https://guides.github.com/activities/forking/)** and clone it locally. Connect your local to the original "upstream" repository by adding it as a remote. Pull in changes from "upstream" often so that you stay up to date so that when you submit your pull request, merge conflicts will be less likely. (See more detailed instructions [here](https://help.github.com/articles/syncing-a-fork/).)
 * **[Create a branch](https://guides.github.com/introduction/flow/)** for your edits.
-* **Reference any relevant issues** or supporting documentation in your PR (ex. "Closes #37.")
+* **Reference any relevant issues** or supporting documentation in your PR (for example "Closes #37.")
```
Comment (MikeMcQuaid) [inline _articles/en-US/how-to-contribute.md]: Needs a comma

File: _articles/en-US/starting-a-project.md [hunk 4]
```diff
@@ -230,7 +230,7 @@ Pick a name that is easy to remember and, ideally, gives some idea of what the p
 * [Sentry](https://github.com/getsentry/sentry) monitors apps for crash reporting
 * [Thin](https://github.com/macournoyer/thin) is a fast and simple Ruby web server
 
-If you're building upon an existing project, using their name as a prefix can help clarify what your project does (ex. [node-fetch](https://github.com/bitinn/node-fetch) brings `window.fetch` to Node.js).
+If you're building upon an existing project, using their name as a prefix can help clarify what your project does (for example [node-fetch](https://github.com/bitinn/node-fetch) brings `window.fetch` to Node.js).
```
Comment (MikeMcQuaid) [inline _articles/en-US/starting-a-project.md]: Needs a comma

File: _articles/en-US/starting-a-project.md [hunk 5]
```diff
@@ -319,7 +319,7 @@ Ready to open source your project? Here's a checklist to help. Check all the box
 <div class="clearfix mb-4">
   <input type="checkbox" id="cbox7" class="d-block float-left mt-1 mr-2" value="checkbox">
   <label for="cbox7" class="overflow-hidden d-block text-normal">
-    There are no sensitive materials in the revision history, issues, or pull requests (ex. passwords or other non-public information)
+    There are no sensitive materials in the revision history, issues, or pull requests (for example passwords or other non-public information)
```
Comment (MikeMcQuaid) [inline _articles/en-US/starting-a-project.md]: Needs a comma

**Comment by bvhme** [comment]
Done, and done!

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Open Source Guides, @bvhme! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**